ansible-ipi-install/inventory/hosts.sample

```diff
@@ -10,6 +10,9 @@ prov_nic=eno1
 # The public NIC (NIC2) used on all baremetal nodes
 pub_nic=eno2
 
+# provisioning bridge name
```

Comment:
Can we change this comment to:
# (Optional) Set the provisioning bridge name. Default value is 'provisioning'.

Reply:
done